src/movies.js

```diff
@@ -1,16 +1,150 @@
 // Iteration 1: All directors? - Get the array of all directors.
+function getAllDirectors(array) {
+  let directors = array.map((movie) => {
+    return movie.director;
+  });
+  return directors;
+}
+
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
-// Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
+function howManyMovies(array) {
+  let spielbergMovie = array.filter((movie) => {
+    return (
+      movie.director === "Steven Spielberg" &&
+      movie.genre.indexOf("Drama") != -1
```

Comment:
indexOf works fine here, but this method is mostly used when you want to know the index in case it exists. If you only need to know if the element is included, then the method .includes works best as it already returns a boolean.